dmriprep/workflows/dwi/base.py

```diff
@@ -297,7 +303,10 @@ def _bold_reg_suffix(fallback):
         )
 
     sdc_report = pe.Node(
-        SimpleBeforeAfter(before_label="Distorted", after_label="Corrected",),
+        SimpleBeforeAfter(
+            before_label="Distorted",
```

Comment:
The corrected image looks more distorted. Have the labels been swapped?

Reply:
What dataset are you looking at? both?

Reply:
Yup I see it in both

Reply:
Okay, I had a look and I believe this is a problem of the ``sloppy`` mode. The final B-Spline regularizer has very low resolution, and hence the distortions are converted into a very smooth field. It does a decent job to reshape the CC in the middle of the brain, but very bad job at the brain edges where the frequency of the distortion is high.

If you look at ds0001771, the reports are the same as in master, right?
If you look at ds000206, the fieldmap-less correction is pretty bad - here I think the sloppy mode is biting at all fronts. It is also that distortion is just too heavy.

Let's go ahead and try to refine this in the future. I think it is important not to be held up by optimization. There's a lot of integration ahead and all these moving pieces will need to find their right place. If we are too strict with that fitting right now we will not have a chance to look at the whole thing.